### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by jburel**
Replace travis by actions
cc @sbesson 

I think we need to discuss the release strategy,
The action ``svenstaro/upload-release-action`` simplifies greatly the release steps

https://github.com/jburel/zeroc-ice-py-manylinux/actions/runs/396762415

### CODE REVIEW

**Comment by sbesson** [comment]
Related to https://github.com/ome/omero-marshal/pull/70#issuecomment-737086696, is this repository supposed to be the place to build and maintain the Ice wheels for all our supported Python distributions? If so I could certainly see adding Python 3.9 as follow-up.

> I think we need to discuss the release strategy,
> The action svenstaro/upload-release-action simplifies greatly the release steps

Also cross-linking to https://github.com/ome/bioformats/pull/3626#issuecomment-718543084 and https://github.com/ome/zeroc-ice-ubuntu2004/pull/1#discussion_r532521621 where these discussions started. I won't have the capacity to assess the action so happy to trust your best  judgement here with regard to consuming it across our repositories.

**Comment by jburel** [comment]
Good point about 3.9, I will see what I can do

**Comment by jburel** [comment]
https://github.com/jburel/zeroc-ice-py-manylinux/actions/runs/397152886 with Python 3.9 support

**Comment by sbesson** [comment]
The artifacts generated by the last commits were successfully tested in https://github.com/sbesson/omero-marshal/actions/runs/398022550. I assume there is only advantages to use these Portable Linux wheels?

**Comment by manics** [comment]
Main disadvantage is they contain a bundled static openssl library. Fine for testing, not so good for production use since they need to be recompiled to get openssl updates, especially if there's an upstream security fix. Also it means OMERO will be using two openssl libraries, the one bundled in this artifact and the one used by Python.

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/pypa/manylinux2014_x86_64:2020-05-30-73c9395
+FROM quay.io/pypa/manylinux2014_x86_64
```
Comment (manics) [inline Dockerfile]: Is there a tag available to ensure reproducible builds? If necessary we can probably set dependabot to automatically bump it.

**Comment by joshmoore** [comment]
> Main disadvantage is they contain a bundled static openssl library.

This has come up before. Shall we consider deprecating these repos?

**Comment by jburel** [comment]
last commit 
tested at https://github.com/jburel/zeroc-ice-py-manylinux/releases/tag/0.3.2

**Review by sbesson — APPROVED** [review]
No objections to deprecating these wheels with the same reasoning as the one raised in https://github.com/ome/zeroc-ice-py-manylinux/pull/4#issuecomment-737887500. Going further, we will need to collectively decide 1- whether this repository should have a final tag for Python 3.9, 2- more generally the set of Ice repositories we are actively supporting

For reference, a few organizational repositories are using the artifacts produced by this repository, primarily for testing purposes - see https://github.com/search?q=org%3Aome+zeroc-ice-py-manylinux&type=code. For testing, I can see two options:

- maintain wheels for all the combination of platforms/Python versions we need. In that case, the main need might be to support for all Python versions from 3.6 to 3.9 on Ubuntu. Places like https://github.com/search?q=org%3Aome+zeroc-ice-py-travis-bionic&type=code could also be updated
- leave GitHub actions installing Ice on the fly

**Comment by sbesson** [comment]
See https://github.com/ome/zeroc-ice-py-travis-bionic/pull/1#issuecomment-745186481

**Pull request merged**